InternetRadioDevice/InternetRadioHeaded/Package.appxmanifest

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="InternetRadioHeaded" Publisher="CN=herricks" Version="1.0.0.0" />
```

Comment:
The publisher is MSFT, that's from the cert and since Herrick created the app from his dev account that's the one it uses. We don't have a Maker cert afaik